test/test_wifi_dto/test_wifi_dto.cpp

```diff
@@ -6,23 +6,24 @@
 #include "WiFiDTO.h"
 
 /**
- * @brief Test WiFIDTO Class Serialization
+ * @brief 
+ * Test WiFIDTO Class Serialization 
+ * This test in in order to show and proof how to use serialized and deserialized 
+ * methods inside DTO class.
  * 
  */
-void test_serialized_wifi_dto(void) {
-     wifi::DTOConfig wifi_params;  //to serialized object 
+void test_serialized_and_deserialized_wifi_dto(void) {
+     wifi::DTOConfig wifi_params;  //to serialized object  declare and initialized this one to be serialized
      wifi_params.apChannel = 4;
      wifi_params.apMaxConn = 3;
      wifi_params.apPassword = tinystring::String("passwordTest");
      wifi_params.apSSID =     tinystring::String("ssidTest");
      wifi_params.isOpen = false;
      wifi_params.WAP_enabled = true;
      wifi_params.WST_enabled = false;
-
-    std::cout << "---------------->>>>>PASSWORD"<< wifi_params.apPassword.c_str() << std::endl;
-    std::cout << "---------------->>>>>SSID"<< wifi_params.apSSID.c_str() << std::endl;
-    std::cout << "------------------------WIFI_PARMS WORKING AS A SERILIZER------------------------\n\n\n\n" << std::endl;
+    
     wifi::DTOConfig wifi_params2; //to deserialized and  retrieve the information 
+    
     //std::ostringstream blob;
     std::stringstream blob;
     std::cout << "------------------------going to Serialized------------------------------" << std::endl;
```

Comment:
Remove `std::cout` from debug code, they are ok for testing in your own branch but for the code inside the `dev` (actually development code intended for real use) it might not be appropiate